src/common/eth-providers/consensus-provider/consensus-provider.service.ts

```diff
@@ -31,7 +31,7 @@ interface RequestRetryOptions {
 
 @Injectable()
 export class ConsensusProviderService {
-  protected rpcUrls: { main: string; backup: string };
+  protected rpcUrls: string[];
```

Comment:
Will be nice to change to something like `apiUrls` in upcoming releases.

Reply:
Renamed all references